### PR TITLE
LPS-189043 | Master

### PR DESCRIPTION
**Pull request by Hanlf**
https://liferay.atlassian.net/browse/LPS-189043

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/dxp/apps/antivirus/antivirus-clamd-scanner-test/testFunctional/tests/ClamdScanner.testcase [hunk 1]
```diff
@@ -22,14 +26,188 @@ definition {
...
+
+		Alert.viewSuccessMessage();
+
+		DepotNavigator.openToDMEntryInDepot(
+			depotName = "Test Depot Name",
+			dmDocumentTitle = "DM Document Title");
+
+		AssertElementNotPresent(
+			key_text = "Download",
+			locator1 = "Button#ANY");
```
Comment (austinchiang) [inline modules/dxp/apps/antivirus/antivirus-clamd-scanner-test/testFunctional/tests/ClamdScanner.testcase:108]: Don't need these steps

File: modules/dxp/apps/antivirus/antivirus-clamd-scanner-test/testFunctional/tests/ClamdScanner.testcase [hunk 2]
```diff
@@ -22,14 +26,188 @@ definition {
...
+		PortletEntry.publish();
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+
+		DERenderer.assertUploadDataIsEmpty();
+	}
+
+	@description = "Verify Clamd Antivirus will block a document that contains a virus uploaded to DM while editing."
+	@priority = 5
+	test CannotUploadVirusDocumentWhileEdiding {
```
Comment (austinchiang) [inline modules/dxp/apps/antivirus/antivirus-clamd-scanner-test/testFunctional/tests/ClamdScanner.testcase:232]: WhileEditing

**Comment by austinchiang** [comment]
@Hanlf for all the test names please use "Virus" instead of "VirusDocument" to keep the test names more succinct. Thx

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please make changes and resend, thx

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
LGTM

**Comment by austinchiang** [comment]
ci:forward

**Comment by Hanlf** [comment]
ci:forward

**Comment by Hanlf** [comment]
ci:test:stable

**Comment by Hanlf** [comment]
ci:forward:force

**Pull request closed without merging**